### PR TITLE
Added recovery endpoints

### PR DESCRIPTION
**Pull request by yuhanniu**

### CODE REVIEW

File: lib/src/main/java/io/token/TokenIO.java [hunk 1]
```diff
@@ -254,6 +258,67 @@ public NotifyStatus notifyPaymentRequest(TokenPayload tokenPayload) {
...
+
+    /**
+     * Completes account recovery.
+     *
+     * @param memberId the member id
+     * @param verificationId the verification id
+     * @param code the code
+     * @return the new member
+     */
+    public Member completeRecovery(
```
Comment (ab666) [inline lib/src/main/java/io/token/TokenIO.java]: rename

File: lib/src/main/java/io/token/TokenIO.java [hunk 2]
```diff
@@ -254,6 +258,67 @@ public NotifyStatus notifyPaymentRequest(TokenPayload tokenPayload) {
...
+
+    /**
+     * Gets recovery authorization from Token.
+     *
+     * @param verificationId the verification id
+     * @param code the code
+     * @param key the privileged key
+     * @return the member recovery operation
+     */
+    public MemberRecoveryOperation getRecoveryAuthorization(
```
Comment (ab666) [inline lib/src/main/java/io/token/TokenIO.java]: maybe rename?

File: lib/src/main/java/io/token/rpc/Client.java [hunk 3]
```diff
@@ -1148,6 +1152,37 @@ public String apply(RetryVerificationResponse response) {
...
+     */
+    public Signature authorizeRecovery(Authorization authorization) {
+        Signer signer = crypto.createSigner(PRIVILEGED);
+        return Signature.newBuilder()
+                .setMemberId(memberId)
+                .setKeyId(signer.getKeyId())
+                .setSignature(signer.sign(authorization))
+                .build();
+    }
+
```
Comment (ab666) [inline lib/src/main/java/io/token/rpc/Client.java]: delete

File: lib/src/main/java/io/token/rpc/UnauthenticatedClient.java [hunk 4]
```diff
@@ -284,4 +301,190 @@ public NotifyStatus apply(RequestTransferResponse response) {
...
+                        .build()))
+                .flatMap(new Function<CompleteRecoveryResponse, Observable<MemberUpdate>>() {
+                    public Observable<MemberUpdate> apply(final CompleteRecoveryResponse res) {
+                        return toObservable(gateway
+                                .getMember(GetMemberRequest.newBuilder()
+                                        .setMemberId(memberId)
+                                        .build()))
+                                .map(new Function<GetMemberResponse, MemberUpdate>() {
+                                    public MemberUpdate apply(GetMemberResponse memberRes) {
+                                        return MemberUpdate.newBuilder()
```
Comment (ab666) [inline lib/src/main/java/io/token/rpc/UnauthenticatedClient.java]: replace with a private converter?

Comment (yuhanniu) [reply lib/src/main/java/io/token/rpc/UnauthenticatedClient.java]: Added `toMemberOperations`

File: lib/src/main/java/io/token/rpc/Client.java (hunk 3 above)
Comment (yuhanniu) [reply lib/src/main/java/io/token/rpc/Client.java]: deleted one of the empty line

**Pull request merged**